<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreationJournal</class>
 <widget class="QDialog" name="CreationJournal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreationJournal</string>
  </property>
  <widget class="QLabel" name="auteur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de l'auteur du Journal  :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Titre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre du journal :</string>
   </property>
  </widget>
  <widget class="QLabel" name="revueJournal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Le nom de la revue du journal :</string>
   </property>
  </widget>
  <widget class="QLabel" name="ISSN_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L'identifiant du journal (ISSN):</string>
   </property>
  </widget>
  <widget class="QLabel" name="Annee">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L'année d'édition du journal :</string>
   </property>
  </widget>
  <widget class="QLabel" name="volume">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Le volume du journal :</string>
   </property>
  </widget>
  <widget class="QLabel" name="numeroJournal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Le numéro du journal :</string>
   </property>
  </widget>
  <widget class="QLabel" name="pageJournal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>La page ou commence la reference:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bienvenue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pour ajouter un journal , veuillez saisir les informations suivantes.</string>
   </property>
  </widget>
  <widget class="QLabel" name="QuitterMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Une fois que vous avez fini de saisir toutes les informations,  veuillez cliquer sur ok.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>650</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="FenetreTitre">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Asssistance pour ajout d'un journal. </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NomAuteurSaisi">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TitreJournalSaisi">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="RevueSaisi">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ISSNSaisi">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Fermer">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>650</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Fermer la fenetre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AnneeSaisie">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="volumeSaisi">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="numeroSaisi">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>490</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PageSaisie">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>560</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>CreationJournal</receiver>
   <slot>Enregistrement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>673</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Fermer</sender>
   <signal>clicked()</signal>
   <receiver>CreationJournal</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>673</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Enregistrement()</slot>
 </slots>
</ui>
